test/publish/test_publication.py

```diff
@@ -178,7 +178,7 @@ def test_write_contributions_by_school(test_session, snapshot, dataset, caplog):
     assert len(df) == 3
 
     # sort it so we know what is in each row
-    df = df.sort_values(["doi"])
```

Comment:
ordering by the primary school too means we can expect the results in a particular order. adding the join to Funders caused the order to fall out of alignment with the testing.